src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -168,7 +163,7 @@ const SignOrExecuteForm = ({
     }
   }
 
-  const onAdvancedSubmit = async (data: AdvancedParameters) => {
+  const onFormSubmit = async (data: AdvancedParameters) => {
```

Comment:
This shouldn't be renamed because there are two forms in this component and the name differentiates them.